src/ecsTransformer.ts

```diff
@@ -46,13 +46,9 @@ export const ecsTransformer = (
 
   const parseClient = () => {
     const req = ctx?.request;
-    let ip;
+    const ip = ctx?.request.header["x-real-ip"] as string | undefined;
```

Comment:
```suggestion
    const ip = req?.header["x-real-ip"] as string | undefined;
```

그리고 x-real-ip 없으면 x-forwareded-for 에 있는 값이라도 분해해서 넣어줘야 하지 않을까요
https://developer.mozilla.org/ko/docs/Web/HTTP/Headers/X-Forwarded-For

Reply:
근데 x-forwarded-for가 있는데 x-real-ip가 없는 경우가 있을까요? x-real-ip는 직전 client ip를 바라보니까요.

Reply:
어차피 헤더는 제거하든 변경하든 추가하든 조작 가능하니까 그냥 있는거 없는거 최대한 보여주자는 취지입니다

Reply:
> 그러면 진짜로 로컬 to 로컬 http request 한건지 아닌지 알아내기가 어려우니까 정보를 알아낼수 없으면 필드도 없애는게 낫지 않을까요

job 같은 경우 때문에 말씀드리긴한건데 local to local인지 http 요청인지 구분이 안되긴합니다.
명확하게 구분할려면 client ip 의 유무로 판단하는게 간단하니 없애는 방향으로 하겠습니다.

Reply:
https://github.com/neekware/fullerstack/tree/main/libs/nax-ipware 이런것도 있긴 하네요,,,